### PR TITLE
.Net: Anthropic - samples

### PR DESCRIPTION
**Pull request by Krzysztof318**
#5690 

Added samples

@RogerBarreto

### CODE REVIEW

**Comment by RogerBarreto** [comment]
@Krzysztof318, thanks for the contribution!

Please update the `README.md` with the link to your new COncepts.

I also suggest to add your Anthropic Connector as one new connector for the `AIRouterDemo` project within Demos.

File: dotnet/samples/Concepts/ChatCompletion/Anthropic_ChatCompletion.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+{
+    [Fact]
+    public async Task SampleAsync()
+    {
+        Console.WriteLine("============= Anthropic - Claude Chat Completion =============");
+
+        string apiKey = TestConfiguration.AnthropicAI.ApiKey;
+        string modelId = TestConfiguration.AnthropicAI.ModelId;
+
+        if (apiKey is null || modelId is null)
```
Comment (RogerBarreto) [inline dotnet/samples/Concepts/ChatCompletion/Anthropic_ChatCompletion.cs:18]: Use `Assert.NotNull` instead.

**Review by RogerBarreto — APPROVED** [review]

**Comment by RogerBarreto** [comment]
@Krzysztof318 

https://github.com/microsoft/semantic-kernel/tree/main/dotnet/samples/Demos/AIModelRouter

**Comment by Krzysztof318** [comment]
Thanks, I had such issue with branch, so that's why I couldn't find that.

File: dotnet/samples/Demos/AIModelRouter/Program.cs [hunk 2]
```diff
@@ -43,7 +44,7 @@ private static async Task Main(string[] args)
             // Find the best service to use based on the user's input
             KernelArguments arguments = new(new PromptExecutionSettings()
             {
-                ServiceId = router.FindService(userMessage, ["lmstudio", "ollama", "openai"])
+                ServiceId = router.FindService(userMessage, ["lmstudio", "ollama", "openai", "claude"])
```
Comment (RogerBarreto) [inline dotnet/samples/Demos/AIModelRouter/Program.cs:47]:
```suggestion
                ServiceId = router.FindService(userMessage, ["lmstudio", "ollama", "openai", "anthropic"])
```

File: dotnet/samples/Demos/AIModelRouter/README.md [hunk 3]
```diff
@@ -23,6 +23,9 @@ The sample can be configured by using the command line with .NET [Secret Manager
 ```powershell
 # OpenAI (Not required if using Azure OpenAI)
 dotnet user-secrets set "OpenAI:ApiKey" "... your api key ... "
+
+# Anthropic
+dotnet user-secrets set "Anthropic:ApiKey" "... your api key ... "
```
Comment (RogerBarreto) [inline dotnet/samples/Demos/AIModelRouter/README.md:28]: Missing the example.

Comment (Krzysztof318) [reply dotnet/samples/Demos/AIModelRouter/README.md:28]: ?

**Pull request merged**